src/services/ApiUsers.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  UserApi,
+  UserControllerGetUserRoleRequest,
+  UserControllerGetUserProfileRequest,
+  UserControllerCreateUserRequest,
+  UserControllerGetMultipleUsersRequest,
+  UserControllerUpdateUserProfileRequest,
+} from './api/apis/UserApi';
+import {
+  AppUserPostRequest,
+  AppUserResponse,
+  AppUserRolesResponse,
+  AppUserProfileResponse,
+  MultipleAppUserResponse,
+  MultipleUserNameResponse,
+  AppUserNameResponse,
+} from './api/models';
+import { Configuration } from './api/runtime';
+import ApiConfigStore from './ApiConfigStore';
+
+export async function getUserProfile(
+  userID: number
+): Promise<AppUserProfileResponse> {
+  const apiConfig: Configuration = ApiConfigStore.getApiConfig();
+  const userApi = new UserApi(apiConfig);
+  const request: UserControllerGetUserProfileRequest = {
+    userID,
+  };
+
+  return userApi.userControllerGetUserProfile(request);
+}
+
+export async function getMultipleUsers(
+  queryParams: UserControllerGetMultipleUsersRequest
+): Promise<MultipleAppUserResponse | MultipleUserNameResponse> {
+  const apiConfig: Configuration = ApiConfigStore.getApiConfig();
+  const userApi = new UserApi(apiConfig);
+
+  return userApi.userControllerGetMultipleUsers(queryParams);
+}
+
+export function processMultipleUsers(
```

Comment:
not sure what this function does, could you explain why we need this? maybe I'm just not following the code

Reply:
Honestly since getMultipleUsers returns a Promise with a union type, I had to make it so that the returned array is only of one of the two types from the union type, or else I couldn't use .map(). Of course, that was before changes made to what we receive as a response if we wanted to have officer names. I will have to either rewrite this in another PR. As of now, think of it as a function that extracts the `users` array from the returned response of type MultipleAppUserResponse.

Reply:
ok - can you call it either getUsernames from response or add a TODO? ideally having an issue would be nice too :)

Reply:
Yeah getUsernames is fine. I'll be making minor fixes/changes to backend tonight as well so this function hopefully will be cleaned up by tonight or tmr.